src/routes/+layout.svelte

```diff
@@ -1,28 +1,38 @@
 <script>
   import '../app.css';
+  import Timeline from '$lib/components/Timeline.svelte';
+  import CommitDrawer from '$lib/components/CommitDrawer.svelte';
+  import { VersionControl } from '$lib/versionControl';
+
+  const vcs = new VersionControl('src/lib/crc-decks/library.json');
+  let commitHistory = 0;
+  let isDrawerOpen = false;
 </script>
 
+<CommitDrawer {isDrawerOpen} {commitHistory} {vcs} />
```

Comment:
I think we're heading in the right direction here, we may need to reintroduce a toolbar header.